### PR TITLE
Match low-hanging fruit

### PR DESCRIPTION
**Pull request by nanaian**
Note not all the no-op funcs identified necessarily take no args.

### CODE REVIEW

File: src/code_f270_len_1190.c [hunk 1]
```diff
@@ -12,7 +12,8 @@ INCLUDE_ASM(s32, "code_f270_len_1190", begin_state_pause_menu);
 
 INCLUDE_ASM(s32, "code_f270_len_1190", step_pause_menu);
 
-INCLUDE_ASM(s32, "code_f270_len_1190", func_80034BF4);
+//INCLUDE_ASM(s32, "code_f270_len_1190", func_80034BF4);
```
Comment (ethteck) [inline src/code_f270_len_1190.c:15]: This comment can be removed

**Review by ethteck — COMMENTED** [review]
I think we should stick with closing braces on the next line instead of {}, but other than that, looks good

**Review by ethteck — APPROVED** [review]

File: src/code_107830_len_e70.c [hunk 2]
```diff
@@ -8,7 +8,8 @@ INCLUDE_ASM(s32, "code_107830_len_e70", func_802E6118);
 
 INCLUDE_ASM(s32, "code_107830_len_e70", func_802E6178);
 
-INCLUDE_ASM(s32, "code_107830_len_e70", func_802E6194);
+void func_802E6194(void) {
+}
```
Comment (pixel-stuck) [inline src/code_107830_len_e70.c:12]: The code quality here is absolutely horrendous, you should be ashamed.

Comment (nanaian) [reply src/code_107830_len_e70.c:12]: Thank you

**Review by pixel-stuck — APPROVED** [review]

**Pull request merged**